rdrf/rdrf/management/commands/set_registry_version.py

```diff
@@ -6,10 +6,14 @@ class Command(BaseCommand):
     help = "Set the registry version"
 
     def handle(self, *args, **options):
-        print("\nATTENTION: You are about to change the version of Registry")
-        for registry in Registry.objects.all():
-            current_version = registry.version or "Not set"
-            print(f"\nCurrent version of {registry.name} is: {current_version}")
-            new_version = input("Enter new version: ")
-            registry.version = new_version
-            registry.save()
+        print("\nThe current registry versions:\n")
+
+        registries = Registry.objects.all().values("name", "code", "version").order_by("code")
```

Comment:
Better to use same non-interactive pattern ( option parser) as for the other commands.

Reply:
modified as suggested